babel.config.js

```diff
@@ -47,6 +47,12 @@ module.exports = function(api) {
           loose: true
         }
       ],
+      [
+        '@babel/plugin-proposal-private-methods',
+        {
+          loose: true,
+        },
```

Comment:
VSCode auto-format on save! You have no power here! Bwaha!
```suggestion
          loose: true
        }
```